examples/tutorials/07_vpp-connection/main.go

```diff
@@ -152,10 +152,11 @@ func (p *HelloWorld) Close() error {
 	return nil
 }
 
-func macParser(mac string) []byte {
+func parseMac(mac string) (parsedMac [6]uint8) {
```

Comment:
This should probably be replaced with generated helper functions for parsing MAC strings.

Reply:
Fixed in https://github.com/ligato/vpp-agent/pull/1785/commits/533ad8fb93433cfe8a9d6d482365be935e58fea9